apps/src/templates/VerticalImageResourceCard.jsx

```diff
@@ -89,7 +90,21 @@ class VerticalImageResourceCard extends Component {
     image: PropTypes.string.isRequired,
     isRtl: PropTypes.bool.isRequired,
     MCShareLink: PropTypes.string,
-    jumbo: PropTypes.bool
+    jumbo: PropTypes.bool,
+    // For logging
+    cardIds: PropTypes.array
+  };
+
+  logToFirehose = (eventName, overrideData = {}) => {
```

Comment:
both of the args here can be removed 🎉 
```suggestion
  logToFirehose = () => {
```